cosmos/operators/_async/base.py

```diff
@@ -0,0 +1,45 @@
+from abc import ABCMeta
+from typing import Any, Sequence
+
+from airflow.utils.context import Context
+
+from cosmos.config import ProfileConfig
+from cosmos.exceptions import CosmosValueError
+from cosmos.operators._async.bigquery import DbtRunAirflowAsyncBigqueryOperator
+from cosmos.operators.local import DbtRunLocalOperator
+
+# Register async operator here
+ASYNC_CLASS_MAP = {"bigquery": DbtRunAirflowAsyncBigqueryOperator}
```

Comment:
@pankajastro it would be great if we could find things more dynamically, like we did in:

https://github.com/astronomer/astronomer-cosmos/blob/7467e1139bd59e831c1a30afee74abea76a03b3e/cosmos/airflow/graph.py#L31-L56

Reply:
Addressed this in commit: f6f2e19edad9a4f4210d89637a76b25d23f6ceea